src/wizards/handoff-banner/style.scss

```diff
@@ -7,10 +7,10 @@
 
 #newspack-handoff-banner {
 	// Color
-	--wp-admin-theme-color: #{colors.$primary-500};
-	--wp-admin-theme-color--rgb: #{colors.$primary-500--rgb};
-	--wp-admin-theme-color-darker-10: #{colors.$primary-600};
-	--wp-admin-theme-color-darker-10--rgb: #{colors.$primary-600--rgb};
+	--wp-admin-theme-color: var(--newspack-ui-color-primary);
+	--wp-admin-theme-color--rgb: var(--newspack-ui-color-primary-rgb);
+	--wp-admin-theme-color-darker-10: var(--newspack-ui-color-primary);
+	--wp-admin-theme-color-darker-10--rgb: var(--newspack-ui-color-primary-rgb);
 	--wp-admin-theme-color-darker-20: #{colors.$primary-700};
 	--wp-admin-theme-color-darker-20--rgb: #{colors.$primary-700--rgb};
```

Comment:
I think here we need to introduce 2 primary variations:

* primary-darker-10: #1522AF (21 34 175)
* primary-darker-20: #0B0B8D (11 11 141)

Reply:
I need a little more clarity here, where do you want these added. Side note, these colors already exist in `newspack-ui`:
https://github.com/Automattic/newspack-plugin/blob/51cbc0ab26dcb7208762404c8b33a1906a35ce67/src/newspack-ui/scss/variables/_colors.scss#L25-L26

Reply:
Ah cool. Can we then have:
```
--wp-admin-theme-color-darker-10: var(--newspack-ui-color-primary-70);
--wp-admin-theme-color-darker-20: var(--newspack-ui-color-primary-80);
```
As for the RGB version – I don't know if it's even possible – can we do something like:
```
--wp-admin-theme-color-darker-10--rgb: rgb( var(--newspack-ui-color-primary-70) );
--wp-admin-theme-color-darker-20--rgb: rgb( var(--newspack-ui-color-primary-80) );
```
If not, then:
```
--wp-admin-theme-color-darker-10--rgb: #{colors.$primary-700--rgb};
--wp-admin-theme-color-darker-20--rgb: #{colors.$primary-800--rgb};
```

Reply:
Aaah, this makes more sense - thanks for the explanation. I have updated to:
```scss
	--wp-admin-theme-color: #{colors.$primary-600};
	--wp-admin-theme-color--rgb: rgb(#{colors.$primary-600--rgb});
	--wp-admin-theme-color-darker-10: #{colors.$primary-700};
	--wp-admin-theme-color-darker-10--rgb: rgb(#{colors.$primary-700--rgb});
	--wp-admin-theme-color-darker-20: #{colors.$primary-800};
	--wp-admin-theme-color-darker-20--rgb: rgb(#{colors.$primary-800--rgb});
```